app/client/components/helpCentre/helpCentreTopic.tsx

```diff
@@ -0,0 +1,91 @@
+import { css } from "@emotion/core";
+import { space } from "@guardian/src-foundations";
+import { brand, neutral } from "@guardian/src-foundations/palette";
+import { headline } from "@guardian/src-foundations/typography";
+import { RouteComponentProps } from "@reach/router";
+import React from "react";
+import { Button } from "../buttons";
+import { SectionContent } from "../sectionContent";
+import { SectionHeader } from "../sectionHeader";
+import { helpCentreConfig, helpCentreNavConfig } from "./helpCentreConfig";
+import { HelpCentreMoreTopics } from "./helpCentreMoreTopics";
+import HelpCentreSingleTopic from "./helpCentreSingleTopic";
+
+interface HelpCentreTopicProps extends RouteComponentProps {
+  topicCode?: string;
+}
+
+const h2Css = css`
+  margin: 0 ${space[3]}px ${space[9]}px ${space[3]}px;
+  border-top: 1px solid ${neutral["86"]};
+  ${headline.small({ fontWeight: "bold" })}
+`;
+
+const divCss = css`
+  margin: 67px ${space[3]}px 0 ${space[3]}px;
+`;
+
+const buttonDivCss = css`
+  margin-top: ${space[4]}px;
+  padding-top: ${space[4]}px;
+  border-top: 1px solid ${neutral["86"]};
+`;
+
+const HelpCentreTopic = (props: HelpCentreTopicProps) => {
+  const selectedNavTopic = helpCentreNavConfig.find(
+    topic => topic.id === props.topicCode
+  );
+
+  const selectedHelpCentreConfigTopic = helpCentreConfig.find(
```

Comment:
Just a reminder from our conversation yesterday. `selectedTopic` would suffice.

Reply:
I suppose with changing this to the if blocks it's clear enough what `selectedTopic` would be now and that this is only temporary code until the config is moved over to s3